tests/main_test.rs

```diff
@@ -1,14 +1,18 @@
+extern crate tempdir;
```

Comment:
is there a reason this is still using the old edition imports?

```suggestion
use tempdir;
```